scripts/src/components.ts

```diff
@@ -8,6 +8,8 @@ import * as descriptors from "./common/descriptors";
 
 import { ConnectorBuilder } from "./connector";
 import * as server from "./server";
+import "./encodeurl";
+import encodeUrl = require('encodeurl')
```

Comment:
Please use double quotes and end with a semicolon.
Yeah, I know, we need lint rules.